### PR TITLE
For #5093, #5076, #5094, #4935: updates to main menu and refactoring of UI tests

### PR DESCRIPTION
**Pull request by sv-ohorvath**
For #5093, #5076: update new menu in UI tests
For #5094: refactor SwitchLocaleTest.kt and fix missing string for "Settings"
For #4935: wait for URL to be updated

### CODE REVIEW

**Comment by isabelrios** [comment]
@sv-ohorvath looks like there is a UI test failure, I can't see the project on Firebase so not sure if the failure is related to any of these tests or a flaky one

**Comment by isabelrios** [comment]
Actually one of the tests modified here failed: [testCustomTabUI](https://console.firebase.google.com/u/0/project/moz-fx-mobile-firebase-testlab/testlab/histories/bh.2b4ca2c7fdb2f0f0/matrices/7825679596468561281/executions/bs.b33a062a2002975f/test-cases)

File: app/src/androidTest/java/org/mozilla/focus/activity/SwitchLocaleTest.kt [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
+    private val frenchLocaleinEn = "Français"
+    private val frenchLanguageMenu = "Langue"
+    private val frenchMotto = "Navigation privée automatique.\nNaviguez. Effacez. Recommencez."
+    private val frenchSettings = "Paramètres"
+    private val frenchHelp = "Aide"
+    private val frenchGeneralHeading = "Général"
+    private val frenchSystemLocale = "Valeur par défaut du système"
+    private val englishLanguageMenu = "Language"
+    private val englishLocaleInFr = "English (United States)"
+
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/focus/activity/SwitchLocaleTest.kt:32]: Is there any other way to access these values? I'm a tad hesitant to be defining these in a test, but don't let this block from seeing these green for now.

**Review by AaronMT — APPROVED** [review]

**Comment by sv-ohorvath** [comment]
> Actually one of the tests modified here failed: [testCustomTabUI](https://console.firebase.google.com/u/0/project/moz-fx-mobile-firebase-testlab/testlab/histories/bh.2b4ca2c7fdb2f0f0/matrices/7825679596468561281/executions/bs.b33a062a2002975f/test-cases)

Yeah, this will fail until https://github.com/mozilla-mobile/focus-android/pull/5089 and https://github.com/mozilla-mobile/focus-android/issues/5084 are solved.

**Comment by isabelrios** [comment]
> > Actually one of the tests modified here failed: [testCustomTabUI](https://console.firebase.google.com/u/0/project/moz-fx-mobile-firebase-testlab/testlab/histories/bh.2b4ca2c7fdb2f0f0/matrices/7825679596468561281/executions/bs.b33a062a2002975f/test-cases)
> 
> Yeah, this one will fail until #5089 and #5084 are solved.

then we should add it commented or it will not block development although it fails?

File: app/src/androidTest/java/org/mozilla/focus/activity/SwitchLocaleTest.kt (hunk 1 above)
Comment (sv-ohorvath) [reply app/src/androidTest/java/org/mozilla/focus/activity/SwitchLocaleTest.kt]: Moved these to a StringsHelper, hope that's ok.

**Pull request merged**